openmls/src/key_packages/mod.rs

```diff
@@ -130,7 +130,40 @@ struct KeyPackageTBS {
     init_key: HpkePublicKey,
     leaf_node: LeafNode,
     credential: Credential, // TODO[FK]: remove
-    extensions: Vec<Extension>,
+    extensions: Vec<KeyPackageExtension>,
```

Comment:
I could revert this to use Extension. The dummy tls_codec impls would go then. If there will be an extension in the future, we could use `#[derive(Tls...)]`